free-1.md

```diff
@@ -58,11 +117,31 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+I don't think that there would be many systems aspects of the language, as the
+overall concept of crochet pattern creations is creative, there would not be
+too many limitations. I think there would be lots of space to work on the design
+both in the language creation and in the two output options.
+
 ### Scope
 
 _How big an idea is this? How ambitious is this project?_
 
+I think it depends on how much work is put into the interactive aspect of the
+project. I imagine that someone could definitely create a very clean language
+and the PDF output. There would then be room for the person to add the
+interactive output, and maybe add more ways for the user to be creative in their
+pattern creation (like adding new stitch definitions).
+
 ### Benefits and drawbacks
 
 _Why might this be a good idea for a project? Why might this not be a good idea
 project?_
+
+I think it would be a good idea because the creator would get to do a variety
+of different things, and the domain is already well defined with it's own
+jargon and rules. Additionally, I was not able to find a program out there that
+can be used to create crochet patterns seamlessly.
+
+If the creator did not already understand what is and isn't possible when
+crocheting, it would be difficult for them to decide on what the language's
```

Comment:
Another reason this would be a good idea is because a good proportion of Mudders like crafts, so you have an audience that you can ask for feedback from. Worst case scenario, you can camp out the Makerspace.